snapcraft/plugins/ant.py

```diff
@@ -38,21 +53,52 @@
 
 class AntPlugin(snapcraft.plugins.jdk.JdkPlugin):
 
+    @classmethod
+    def schema(cls):
+        schema = super().schema()
+        schema['properties']['ant-properties'] = {
+            'type': 'object',
+            'default': {},
+        }
+        schema['properties']['ant-build-target'] = {
+            'type': 'string',
+        }
+        schema['properties']['ant-dest-property'] = {
```

Comment:
The others are prefect, just wondering if `ant-dest-property` was just to get non exposed information from the plugin.

In any case, https://github.com/snapcore/snapcraft/pull/757 (variable naming pending a discussion)